pkg/katautils/katatrace/tracing.go

```diff
@@ -90,19 +103,32 @@ func StopTracing(ctx context.Context) {
 
 // Trace creates a new tracing span based on the specified name and parent
 // context.
-func Trace(parent context.Context, name string) (opentracing.Span, context.Context) {
+func Trace(parent context.Context, logger *logrus.Entry, name string, tags ...string) (opentracing.Span, context.Context) {
```

Comment:
May be worth adding a note in the header re: tags. ie, its a slice of strings, but really its full of key-value pairs, right? Let's just make sure the caller is aware that there should be an even number of entries

Reply:
Fixed